test/change.test.js

```diff
@@ -610,71 +610,66 @@ describe('Change', function() {
 });
 
 describe('Change with with custom properties', function() {
-  var ChangeWithCustomProperty, TestModelWithModifiedChangeModel;
-  var customProperty = 'tenantId';
+  let Change, TestModel;
 
   beforeEach(function() {
-    var memory = loopback.createDataSource({
+    let memory = loopback.createDataSource({
       connector: loopback.Memory,
     });
 
-    var additionalChangeModelProperties = {};
-    additionalChangeModelProperties[customProperty] = {type: 'string'};
-
-    TestModelWithModifiedChangeModel = loopback.PersistedModel
-    .extend('ChangeTestModel',
+    TestModel = loopback.PersistedModel.extend('ChangeTestModelWithTenant',
       {
         id: {id: true, type: 'string', defaultFn: 'guid'},
+        tenantId: 'string',
```

Comment:
This was probably an oversight - the custom property should be defined on both Model and Change, right?